examples/with-multifactorauth/README.md

```diff
@@ -0,0 +1,57 @@
+![SuperTokens banner](https://raw.githubusercontent.com/supertokens/supertokens-logo/master/images/Artboard%20%E2%80%93%2027%402x.png)
+
+# SuperTokens Google one tap Demo app
```

Comment:
And we need to make this demo app very simple. Just have users by default have no MFA, and then in the home page, they can enable MFA (one of totp, email otp or sms otp) and then once they do that, when they login next, it will ask them for that factor. This should be doable using just our mfa recipe functions, and ideally we do not need to manually set user metadata etc.